### DISCUSSION TITLE
Can't disable Update Notifications in Taskbar in Windows XI

### DISCUSSION THREAD

**Discussion by PapaPetro** [valinet/ExplorerPatcher#2062] [General]
Great service.
I just don't like the update popup on Windows startup.
Feels needy and intrusive.
This either should be handled in the background or have the option the remove manual notifications.

Remember, the annoyances of the small stuff is the reason we are using ExplorerPatcher in the first place (to fix the small stuff that Microsoft screwed up with their frequent updates).
Please don't lose sight of this and repeat history...

Also go beyond being a fragile ego and see the bigger picture.

**Comment by Poopooracoocoo** [comment]
https://github.com/valinet/ExplorerPatcher/wiki/Configure-updates

Updates to EP are a good thing. Updates to Windows may not be. The former is made by an independent developer and a handful of one-time contributors. The latter is made by a trillion-dollar megacorp. They're not at all comparable. EP doesn't update very often. And if you're somehow seeing more prompts, something is wrong. Windows doesn't let you opt out of updates and so EP won't always be compatible with the latest GA release of Windows, especially Windows 11, which receives more development than 10. 

>Also go beyond being a fragile ego and see the bigger picture.

lol wat. A feature request for silent/automatic updates would be better than attacking the maintainer for no reason
